packages/eslint-plugin/src/rules/no-unsafe-call.ts

```diff
@@ -20,7 +20,7 @@ export default createRule<[], MessageIds>({
   meta: {
     type: 'problem',
     docs: {
-      description: 'Disallow calling a value with type `any`',
+      description: 'Disallow calling a value with type an {{type}} value',
```

Comment:
...in other words, you odn't need to update this string. `docs.description` is a description of the rule. It doesn't show up in error messages.